src/shared/cache/Grch38GenomeNexusCache.ts

```diff
@@ -0,0 +1,49 @@
+import { fetchVariantAnnotationsByMutation } from 'shared/lib/StoreUtils';
```

Comment:
Do we really need to implement a whole new cache class? Can't we modify the existing one to make it compatible with different URLs? This is mostly copy/paste from the original `GenomeNexusCahce`, right?

Reply:
Had a discussion about it. We need to use different urls in the fetch function, but fetch function can’t accept other urls, since the GenomeNexusCache is passing the fetch as a function to LazyMobXCache. Ideally we should be able to pass URL to the constructor and use that in the fetch function, but it might be too much work to achieve that at this point

Reply:
Temporary fix we had now is to pass a fetch function from store.